src/utils/react.js

```diff
@@ -0,0 +1,41 @@
+/* eslint prefer-destructuring: 0 */
+// @flow
+import * as React from 'react';
+
+/**
+ * Type from:
+ * https://github.com/facebook/react/blob/master/packages/react/src/ReactHooks.js
+ * https://github.com/streamich/react-use/blob/master/src/react.ts
```

Comment:
Currently `react.ts` is missing in https://github.com/streamich/react-use/tree/master/src

Reply:
Hooks have already been supported in TS so `react-use` removed it.
https://github.com/streamich/react-use/commit/cdc5ec596d216eb63e24b500a1d7a86a0823f214